dojo.py

```diff
@@ -97,3 +97,122 @@ def allocate_room(self, new_person, room):
             living_space.occupants.append(new_person)
             print('{0} has been allocated the living spaces {1}'.format(new_person.person_name, living_space.room_name))
 
+    def print_room(self, room):
+        """ checks if room name given exists in the list of total rooms.
+            if so it prints out a statement that highlights the room name,
+            the room type and occupants.
+        """
+        if self.total_rooms:
+            if room in [room.room_name for room in self.total_rooms]:
+                for room_to_check in self.total_rooms:
+                    if room == room_to_check.room_name:
+                        if len(room_to_check.occupants) > 0:
+                            people = []
+                            for occupant in room_to_check.occupants:
+                                people.append(occupant.person_name)
+                            str1 = ', '.join(str(e) for e in people)
+                            print(
```

Comment:
Another thing, avoid using letters like e for variables, because when you need to search for "e" in your file, it will take you ages.